tests/test_sequence_encodings.py

```diff
@@ -97,14 +105,39 @@ def test_botvinnikize(peptide_string):
     assert test == true
 
 
-def test_encode_peptide(peptide_string, molecule):
+def test_PEPTIDE_MAPPINGS():
```

Comment:
def test_peptide_constants. usually not good to have capital/big letters in test/function names